### PR TITLE
DT-738:  amended backup email pages to match prototype and reduced al…

### PR DESCRIPTION
**Pull request by SimonMitchellMOJ**
…l headings from xl to l

### CODE REVIEW

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyEmailControllerTest.kt [hunk 1]
```diff
@@ -69,12 +69,12 @@ class VerifyEmailControllerTest {
     verify(jwtAuthenticationSuccessHandler).proceed(request, response, principal)
   }
 
-  @Test
-  fun verifyEmailContinue() {
-    SecurityContextHolder.getContext().authentication = principal
-    verifyEmailController.verifyEmailContinue(request, response)
-    verify(jwtAuthenticationSuccessHandler).proceed(request, response, principal)
-  }
+//  @Test
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyEmailControllerTest.kt:72]: commented out code? Just remove

File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyEmailController.java [hunk 2]
```diff
@@ -79,11 +79,6 @@ public ModelAndView verifyEmailRequest(final Principal principal, final HttpServ
         return modelAndView;
     }
 
-    @GetMapping("/verify-email-continue")
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyEmailController.java:82]: why has this gone?

Comment (petergphillips) [reply src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyEmailController.java:82]: what about the primary flow when they are interrupted on login due to not verifying their email address?

Comment (SimonMitchellMOJ) [reply src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyEmailController.java:82]: that process uses a difference endpoint to skip - `@GetMapping("/verify-email-skip")`, all the `@GetMapping("/verify-email-continue")` did was proceed to the homepage

**Pull request merged**